installation/configuration.md

```diff
@@ -1,112 +1,133 @@
 # Configuration
-The config file allows you to set the basic configuration for database, theme, email and file handling
 
-## Theme Configuration
-Leantime allows you to white label the entire system by changing the color scheme of your site and replacing the logo. 
-Just set the value `mainColor` to your primary color and `logoPath`. 
-
-These settings can also be changed within Leantime under Account->Company Settings.
+Leantime can be configured to setup database, customize theme, email and file handling:
 
 ## S3 Configuration
-Leantime can store your files on the local file system or in AWS S3. To get started with S3 you will need a bucket in the region of your choice and an account with permissions to write objects to yout bucket. You should have an account key and secret as well as your region ready.
 
-In your configuration.php edit the following lines:
+Leantime has flexible file storage option. You can choose between local file storage or Amazon S3. Here's how to set up S3 storage:
```

Comment:
@marcelfolaron thoughts on adding how to update the storage size? We still get the occasional request on how to up the limit.